.buildkite/rllib.rayci.yml

```diff
@@ -67,30 +46,38 @@ steps:
       - bazel run //ci/ray_ci:test_in_docker -- //rllib/... rllib
         --workers "$${BUILDKITE_PARALLEL_JOB_COUNT}" --worker-id "$${BUILDKITE_PARALLEL_JOB}" --parallelism-per-worker 2
         --only-tags examples
-        --except-tags multi_gpu,gpu,examples_use_all_core
+        --except-tags gpu,multi_gpu,manual,examples_use_all_core
         --test-env RAY_USE_MULTIPROCESSING_CPU_COUNT=1
         --build-name rllibbuild-py3.10
         --python-version 3.10
       - bazel run //ci/ray_ci:test_in_docker -- //rllib/... rllib
         --workers "$${BUILDKITE_PARALLEL_JOB_COUNT}" --worker-id "$${BUILDKITE_PARALLEL_JOB}"
         --only-tags examples_use_all_core
-        --skip-ray-installation
-        --except-tags multi_gpu,gpu
+        --except-tags gpu,multi_gpu,manual
         --test-env RAY_USE_MULTIPROCESSING_CPU_COUNT=1
         --build-name rllibbuild-py3.10
         --python-version 3.10
+        --skip-ray-installation
     depends_on: rllibbuild
 
-  - label: ":brain: rllib: tests dir"
-    tags: rllib_directly
-    parallelism: 2
+  - label: ":brain: rllib: learning tests"
+    tags: rllib
+    parallelism: 5
     instance_type: large
     commands:
       - bazel run //ci/ray_ci:test_in_docker -- //rllib/... rllib
         --workers "$${BUILDKITE_PARALLEL_JOB_COUNT}" --worker-id "$${BUILDKITE_PARALLEL_JOB}" --parallelism-per-worker 3
-        --only-tags tests_dir
-        --except-tags multi_gpu,manual
-        --test-env RAY_USE_MULTIPROCESSING_CPU_COUNT=1
+        --only-tags learning_tests
+        --except-tags gpu,multi_gpu,learning_tests_pytorch_use_all_core,manual
+        --test-arg --framework=torch
+        --build-name rllibbuild-py3.10
+        --python-version 3.10
+      - bazel run //ci/ray_ci:test_in_docker -- //rllib/... rllib
```

Comment:
This entire `bazel run ...` can be removed if we drop `learning_tests_pytorch_use_all_core` tag.